version.sbt

```diff
@@ -1,12 +1 @@
-import java.io.FileInputStream
-import java.util.Properties
-
-version in ThisBuild := {
-  val s = new FileInputStream(file("version.properties"))
-  try {
-    val p = new Properties()
-    p.load(s)
-    p.getProperty("version")
-  } finally s.close()
-
-}
+version in ThisBuild := "0.26.0-SNAPSHOT"
```

Comment:
@ktoso If you could explain why we went with this property file based approach then that'd be helpful. As you can see what we now have is much simpler.